Source/WarpX.H

```diff
@@ -653,11 +670,22 @@ private:
     int n_buffer = 4;
     amrex::Real const_dt = 0.5e-11;
 
+    // Load balancing
     int load_balance_int = -1;
     amrex::Vector<std::unique_ptr<amrex::MultiFab> > costs;
+    amrex::Vector<std::unique_ptr<amrex::MultiFab> > costs_heuristic;
     int load_balance_with_sfc = 0;
     amrex::Real load_balance_knapsack_factor = 1.24;
 
+    const std::map<std::string, LBType> load_balance_type_str_to_load_balance_type = {
```

Comment:
I would place this map very close to the enum definition, because it might get out of sync on later patches otherwise.

Reply:
Moved to after the `enum` definition.